.github/workflows/go.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.15.x, 1.16.x]
+        go-version: [1.15.x, 1.16.x, 1.17.x, 1.18.x]
```

Comment:
We can drop EOL'd versions (1.15, 1.16) here.

Reply:
@oschwald 
Hi, Thanks for reviewing! I fixed it.